sonarqube/assets/configuration/spec.yaml

```diff
@@ -76,19 +76,26 @@ files:
 
           Note: This is required for all project checks, whether `is_jmx` is set to `true`
           or `false.
```

Comment:
Is this still correct? The ddev JMX instance configs don't have the `components` configured

Reply:
I think that was never necessary